oapen-engine/src/tasks/daemon.py

```diff
@@ -12,10 +18,30 @@ def signal_handler(signal, frame):
 
 
 print("Daemon up and running")
+print("STARTED: clean.py")
+run_clean()
```

Comment:
Should we be running a clean on every daemon startup?

Reply:
Probably not, can maybe have an env variable that flushes it on startup for dev?